src/components/UI/Pagination.tsx

```diff
@@ -0,0 +1,27 @@
+import css from "./Pagination.module.css";
+interface Paginate {
+  paginate: (pageNumber: number) => void;
+}
+const Pagination = ({ paginate }: Paginate) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= 2; i++) {
+    pageNumbers.push(i);
+  }
+
+  return (
+    <nav>
+      <ul className="pagination">
+        {pageNumbers.map((number) => (
+          <li key={number} className={`page-item ${css["pagination-width"]}`}>
+            <a onClick={() => paginate(number)} href="!#" className="page-link">
```

Comment:
Either replace `a` tag with button or remove href prop.
Currently it will hard refresh the browser window. As a result, state will reset for whole app including redux.